hutool-core/src/test/java/cn/hutool/core/stream/EntryStreamTest.java

```diff
@@ -116,7 +125,7 @@ public void testPeekKey() {
 		final List<Integer> keys = new ArrayList<>();
 		EntryStream.of(Arrays.asList(new Entry<>(1, 1), new Entry<>(1, 2), new Entry<>(2, 1), new Entry<>(2, 2)))
 			.peekKey(keys::add)
-			.count();
+			.collect(Collectors.toList());
```

Comment:
没有使用便捷的toList()方法, 差评
```suggestion
			.toList();
```